tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -103,10 +105,21 @@ func (c *cluster) bootstrap() error {
 		}
 	}
 
-	c.Stressers = make([]Stresser, len(members))
+	c.Stressers = make([]Stresser, 0)
```

Comment:
the stresser needs to contain both original stresser and lease stresser. so the the length is not defined.